resources/assets/components/ReportbackUploader/index.js

```diff
@@ -120,8 +120,8 @@ class ReportbackUploader extends React.Component {
 }
 
 ReportbackUploader.propTypes = {
-  campaignId: PropTypes.number.isRequired,
-  campaignRunId: PropTypes.number.isRequired,
+  campaignId: PropTypes.string.isRequired,
```

Comment:
It is odd that `campaignID` and `campaign_run_id` are different types, any reason why?

Reply:
(Idle thought, not something to fix in this PR:) We should really standardize that all IDs are requested and returned as strings, regardless of actual values. We'd made that call a long while back in Ashes, but didn't really do a great job of sticking to it. Otherwise it adds some awkward cognitive overhead when switching between querying different apps.

Reply:
@DFurnes yea I agree, that should just be a normal convention.

Reply:
cool! I added a card [here](https://www.pivotaltracker.com/n/projects/2019429/stories/156552592)! cc @DFurnes @sbsmith86